insights/parsers/auditctl_status.py

```diff
@@ -36,6 +41,14 @@ class AuditctlStatus(LegacyItemAccess, CommandParser):
         >>> auds['enabled']
         1
     """
+    def __init__(self, context):
+        deprecated(
+            AuditctlStatus,
+            "Please use the :class:`insights.parsers.auditctl.AuditdStatus` instead.",
+            "3.1.25"
```

Comment:
@psachin after discussion with @xiangce , we agree to add a new module to include the parsers about "auditctl" commands, because the original file name is "auditctl_status.py", it's not good to add other "auditctl" commands. And I mark the original one as deprecated. Now the version is wrong, I'm not sure which version I should use, please help here, thanks.

Reply:
3.1.25 is OK

Reply:
Better introduce a global variable, e.g. "NEXT_MINOR_VER", to avoid such kind of confusion.

Reply:
Currently, the exact version list isn't done, so a few parsers are marked as "3.1.25". Sachin said it is a release which is one year later. After the version list comes out, maybe we'll mark different versions for the parser. Maybe in one release, there is only one parser deprecated. In this case, maybe a global variable is useless? Do we need to do it now for "3.1.25"?

Reply:
Sorry for the incomplete comment, that is for @psachin, but not a block for this PR, please go ahead and use the hard-coded `3.1.25` .

> Maybe in one release, there is only one parser deprecated. In this case, maybe a global variable is useless?

No matter how many `component` would be deprecated in the next minor release version, it would be better to use a global variable than the hard-coded versions, from the development and release perspective.

Reply:
@xiangce By using custom version the developers can plan the feature deprecation accordingly. The developer might not want to deprecate the feature in NEXT_MINOR_VER. I have drafted a change in the docs with examples to help them in this process[1]. 

[1]  https://github.com/RedHatInsights/insights-core/pull/3500